### DISCUSSION TITLE
Content guidelines?

### DISCUSSION THREAD

**Discussion by jwender** [hpc-social/mastodon-instance-and-policies#5] [General]
When thinking about what to toot, I was wondering what the expectations are on hpc on-topic and off-topic contents? Should mast.hpc.social be focused on hpc and only seldom other things or is it ok if I toot also other stuff?
The question also relates to whether I would need a second account for non-hpc posts?
What are your thoughts?

**Comment by alansill** [comment]
It seems to me to be appropriate to have other accounts on other servers if most of your posts will be off-topic for a particular server. Most Mastodon clients have provisions to switch between accounts. This allows others to decide which account to follow if your posts will be identifiably different from each other for your different accounts. Ideally you'd find a separate server aimed at each type of community that you want to interact with. But many people, if not most, probably don't want to put in the effort to manage too many accounts. There are probably many correct answers and approaches here.

**Comment by vsoch** [comment]
I think I disagree! It's hard to find tools to easily switch between accounts - the only suggestion I've seen is Firefox containers to have them both open at once. I think if you identify strongly with HPC (and would thus choose mast.hpc.social) you should be able to use the account to express all aspects of yourself. It's likely the case that a lot of your toots will be HPC related but I don't see why they have to be.

**Reply by alansill** [reply]
On the web you're correct - one account per browser seems to be the norm (though I have not experimetned with incognito windows etc. But apps built for Mastodon often do have the ability to switch  between accounts. Here's an example from Mastonaut andI think it's a pretty popular feature, though personally I only use one account for routine posting. (I do have a couple of others for administrative purposes but rarely use them, so opening a different browser works for me for those cases.)
![Screen Shot 2022-11-29 at 9 17 31 AM](https://user-images.githubusercontent.com/2364484/204568891-8b31f795-e335-4191-900d-9a74f0ba8561.png)

**Reply by vsoch** [reply]
oh neat! Is this web / iOS / Android / other? It looks like it might just be for Apple? (if this is it?)

https://mastonaut.app/

**Comment by jwender** [comment]
Mastonaut looks like MacOS only ☹ but for instance MetaText for iOS also supports multiple accounts.

**Reply by vsoch** [reply]
Maybe this is why I haven't found any! I'm on Linux boxes! :boxing_glove: :package:
